akka-stream-typed/src/test/scala/docs/akka/stream/typed/ActorSourceSinkExample.scala

```diff
@@ -42,6 +42,71 @@ object ActorSourceSinkExample {
     // #actor-source-ref
   }
 
+  {
+
+    implicit val system: ActorSystem[_] = ???
+
+    // #actor-source-with-backpressure
+    import akka.actor.typed.ActorRef
+    import akka.stream.CompletionStrategy
+    import akka.stream.scaladsl.{ Sink, Source }
+    import akka.stream.typed.scaladsl.ActorSource
+
+    case object Ack
+
+    trait Protocol
+    case class Message(msg: String) extends Protocol
+    case object Complete extends Protocol
+    case class Fail(ex: Exception) extends Protocol
+
+    def sendingActor: Behavior[Ack.type] =
```

Comment:
Align with Akka Typed styleguide, object with apply for creating actor, put messages it accepts nested in it etc.

I think injecting the stream `ActorRef` instead of creating it nested in the sender could separate concerns a bit and make it easier to follow (esp. the Java one)